tox_pdm/plugin4.py

```diff
@@ -58,3 +62,17 @@ def register_config(self) -> None:
     @staticmethod
     def id() -> str:
         return "pdm"
+
+    def execute(
+        self,
+        cmd: t.Sequence[Path | str],
+        stdin: StdinSource,
+        show: bool | None = None,
+        cwd: Path | None = None,
+        run_id: str = "",
+        executor: Execute | None = None,
+    ) -> Outcome:
+        if scripts := pdm_scripts(self.core["tox_root"]):
```

Comment:
`tox-pdm` does support python 3.7, so does `tox4`. We can't use the walrus operator here.

Reply:
Done 👍🏼